pytest_examples/test_class.py

```diff
@@ -2,82 +2,101 @@
 
 from pytest_examples.class_to_test import ClassToTest
 
+from freezegun import freeze_time
+
+import datetime
+
 @pytest.fixture
 def any_fit():
     return ClassToTest()
 
+
+@freeze_time("2021-12-09")
+class TestGetCurrent(ClassToTest):
+    def test_current_time(self):
+        assert datetime.datetime.now() == datetime.datetime(2021, 12, 9)
```

Comment:
I would love a few tests (instead of just one) and for them to be testing the `get_current_time` method instead of just calling datetime.